app/modules.go

```diff
@@ -64,20 +64,19 @@ import (
 
 // module account permissions
 var mAccPerms = map[string][]string{
-	authtypes.FeeCollectorName:                         nil,
-	distrtypes.ModuleName:                              nil,
-	icatypes.ModuleName:                                nil,
-	minttypes.ModuleName:                               {authtypes.Minter},
-	stakingtypes.BondedPoolName:                        {authtypes.Burner, authtypes.Staking},
-	stakingtypes.NotBondedPoolName:                     {authtypes.Burner, authtypes.Staking},
-	govtypes.ModuleName:                                {authtypes.Burner},
-	ibctransfertypes.ModuleName:                        {authtypes.Minter, authtypes.Burner},
-	strategicreservemoduletypes.SRPoolName:             nil,
-	strategicreservemoduletypes.BetReserveName:         nil,
-	strategicreservemoduletypes.SRProfitName:           nil,
-	strategicreservemoduletypes.OrderBookLiquidityName: nil,
-	betmoduletypes.ModuleName:                          nil,
-	housemoduletypes.HouseParticipationFeeName:         nil,
+	authtypes.FeeCollectorName:                        nil,
+	distrtypes.ModuleName:                             nil,
+	icatypes.ModuleName:                               nil,
+	minttypes.ModuleName:                              {authtypes.Minter},
+	stakingtypes.BondedPoolName:                       {authtypes.Burner, authtypes.Staking},
+	stakingtypes.NotBondedPoolName:                    {authtypes.Burner, authtypes.Staking},
+	govtypes.ModuleName:                               {authtypes.Burner},
+	ibctransfertypes.ModuleName:                       {authtypes.Minter, authtypes.Burner},
+	betmoduletypes.BetCollector:                       nil,
+	betmoduletypes.BetFeeCollector:                    nil,
+	housemoduletypes.HouseFeeCollector:                nil,
+	strategicreservemoduletypes.DataFeeCollector:      nil,
```

Comment:
@scorpioborn also the affinity of data fee collector is with the market module because the data which is added to blockchain is into the market module and not strategicreserve.

So please move these 2 module accounts

Reply:
Data Fee Collector removed in #153